src/parachain/collateral.ts

```diff
@@ -26,19 +21,25 @@ export interface CollateralAPI {
      * @returns The free balance of the given account
      */
     balance(id: AccountId): Promise<Big>;
+    /**
+     * Subscribe to free balance updates
+     * @param account AccountId string
+     * @param callback Function to be called whenever the balance of an account is updated.
+     * Its parameters are (accountIdString, freeBalance)
+     */
+     subscribeToBalance(account: string, callback: (account: string, balance: Big) => void): Promise<() => void>;
     /**
      * Send a transaction that transfers from the caller's address to another address
      * @param address The recipient of the transfer
      * @param amount The balance to transfer
```

Comment:
please add "denominated in DOT"

Reply:
Would we specify this in the description of methods or of the encompassing class?

Reply:
On the method, it has the advantage is that you can see the doc comments in VSCode on the function. But it's duplication. What do you think? I think having it in the class should normally suffice and we just stick throughout the code base.

Reply:
Switching to the monetary lib will clarify expected denominations. Since that lib is close to actually being created, I think this duplication won't bring much benefit. I'll mention the denomination here, while in other places I'll just specify in the class description.